tests/run_manual_test_windows.bat

```diff
@@ -1,12 +1,13 @@
 :: this script must be executed from the Test directory
 cd ../
+pip install Flask
```

Comment:
Can you also add "pip install pyreadline3"? Just for the windows bat file is enough

Reply:
Done, exactly as requested.